geonosis/main.go

```diff
@@ -30,8 +32,67 @@ func (t *Template) Render(w io.Writer, name string, data interface{}) error {
 	return t.templates.ExecuteTemplate(w, name, data)
 }
 
+type test_struct struct {
+	Image string
+	Name  string
+	Cmd   []string
+	Env   []string
+}
+
 // Handlers
 func createDeployment(c *echo.Context) error {
+
+	r := c.Request()
+	body, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println(string(body))
+	var t test_struct
```

Comment:
I think that was just the end of a series of desperate tries to get something to work...